pkg/portallocator/doc.go

```diff
@@ -0,0 +1,16 @@
+// Copyright 2018 Google LLC All Rights Reserved.
```

Comment:
nit: This looks like a new file, so we should use the current year for copyright instead of using the year the other files were created.

Reply:
Done